README.md

```diff
@@ -5,11 +5,11 @@
 
 ## Overview
 
-This repository contains configuration files and [overlays](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/#bases-and-overlays) to deploy Kubeflow on [Amazon Elastic Kubernetes Service (Amazon EKS)](https://aws.amazon.com/eks/).
+Kubeflow on AWS is an open source distribution of [Kubeflow](https://www.kubeflow.org/) that provides its own Kubeflow manifests to support integrations with various AWS managed services. Use Kubeflow on AWS to streamline data science tasks and build highly reliable, secure, and scalable machine learning systems with reduced operational overheads.
 
 ## Getting Started
 
-There are a number of deployment options for installing Kubeflow with AWS service integrations. For detailed instructions regarding available AWS-integrated deployment options, see the Deployment section on the [Kubeflow on AWS](https://awslabs.github.io/kubeflow-manifests/) website.
+There are a number of deployment options for installing Kubeflow with AWS service integrations. To get started with deploying Kubeflow on [Amazon Elastic Kubernetes Service (Amazon EKS)](https://aws.amazon.com/eks/), see the [deployment section](https://awslabs.github.io/kubeflow-manifests/docs/deployment/) on the [Kubeflow on AWS](https://awslabs.github.io/kubeflow-manifests/) website.
```

Comment:
remove `deploying `

Reply:
sentence reads strange without it - Kubeflow on EKS, Kubeflow on AWS

To get started with Kubeflow on Amazon EKS, see the deployment section on the Kubeflow on AWS website.
I made this change to include EKS specifically.

other option is - 
To get started, see the deployment section on the Kubeflow on AWS website.

wdyt?

Reply:
I think the sentence can constructed better 
we can do this later 
ok for now